rust-runtime/smithy-json/fuzz/.gitignore

```diff
@@ -0,0 +1,7 @@
+
+target
+corpus
```

Comment:
That was the default from `cargo fuzz`. I force added the seed files, and this ignore should prevent adding any of the libFuzzer generated corpus documents. Whether or not it will be valuable to commit those is open for debate.